src/app/containers/RadioSchedule/index.jsx

```diff
@@ -19,8 +24,14 @@ const RadioScheduleContainer = ({ endpoint }) => {
         .then(handleResponse)
         .catch(e => logger.error(`HTTP Error: "${e}"`));
 
-    fetchRadioScheduleData(endpoint);
-  }, [endpoint]);
+    if (enabled) {
+      fetchRadioScheduleData(endpoint);
+    }
+  }, [endpoint, enabled]);
+
+  if (!enabled || !hasRadioSchedule) {
```

Comment:
I think we should move this logic `ln32 - ln34` right below `ln12`. We don't need to define the other constants if there are not going to be used.

Reply:
I think this would be against one of the Rules of Hooks https://reactjs.org/docs/hooks-rules.html since the useEffect would become conditional. And probably produce a ESLint error.

Reply:
Moving the logic below `ln12` does result in an ESLint as stated above